src/avatar/src/utils/hash.ts

```diff
@@ -1,8 +1,8 @@
 // http://erlycoder.com/49/javascript-hash-functions-to-convert-string-into-integer-hash-
-export default function hashCode(s) {
+export default function hashCode(s: any): number {
```

Comment:
Is it possible to use `s: unknown` here?

Reply:
I can try that out.